editor/src/components/canvas/commands/utils/property-utils.ts

```diff
@@ -99,3 +102,24 @@ export function maybeCssPropertyFromInlineStyle(property: PropertyPath): string
   }
   return prop
 }
+
+export type GetCSSNumberFromStyleInfoResult =
+  | { type: 'not-found' }
+  | { type: 'not-css-number' }
+  | { type: 'css-number'; number: CSSNumber }
+
+export function getCSSNumberFromStyleInfo(
+  styleInfo: StyleInfo,
+  property: string,
+): GetCSSNumberFromStyleInfoResult {
+  let styleInfoUntyped = styleInfo as any as Record<string, CSSStyleProperty<unknown>>
```

Comment:
I believe this can be done without the need for a cast:
```typescript
  if (!isStyleInfoKey(property)) {
    return { type: 'not-found' }
  }

  const prop = styleInfo[property]
  if (prop == null || prop.type === 'not-found') {
    return { type: 'not-found' }
  }

  if (prop.type === 'not-parsable' || !isCSSNumber(prop.value)) {
    return { type: 'not-css-number' }
  }
  return { type: 'css-number', number: prop.value }
```

Reply:
pushed the code without a cast in https://github.com/concrete-utopia/utopia/pull/6632/commits/886103bbb5390c7abbe3fb454fd8291e706e133e